ENIGMAsystem/SHELL/Universal_System/Extensions/XRandR/About.ey

```diff
@@ -0,0 +1,13 @@
+%E-YAML
+---
+
+Name: XRandR
+Identifier: XRandR
+Author: Robert Colton
+Description: Adds Xlib display functions using the X Resize and Rotate extension.
```

Comment:
Please add a note about how changing my display settings is an act of aggression, and if you attempt to do this, *they will  n e v e r   f i n d   y o u r   b o d y.*

Reply:
Ok: 1c47c0f14eab3dcb94ede548d4325698e4802f00

Reply:
You misspelled "never find your body."

Reply:
Fixed that for you. e2bf020fb5ffce9d96f9fe1e68b94efb241c15c8

Reply:
Whatever, fair enough.